e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -216,30 +227,32 @@ describe('Integrations', () => {
         // # Open RHS (reply thread)
         cy.clickPostCommentIcon();
 
-        // # type /me test
-        cy.get('#reply_textbox').type('/me test');
+        const message = getRandomId();
+
+        // # type /me message
+        cy.get('#reply_textbox').type(`/me ${message}`);
         cy.get('#addCommentButton').click();
-        cy.uiWaitUntilMessagePostedIncludes('test');
+        cy.uiWaitUntilMessagePostedIncludes(message);
 
         cy.getLastPostId().then((postId) => {
             // * Verify RHS message is from current user and properly formatted with lower opacity
             cy.get(`#rhsPost_${postId}`).should('have.class', 'current--user').within(() => {
                 cy.get('button').should('have.text', user1.username);
-                cy.get('p').should('have.text', 'test').and('have.css', 'color', 'rgba(61, 60, 64, 0.6)');
+                cy.get('p').should('have.text', message).and('have.css', 'color', 'rgba(61, 60, 64, 0.6)');
             });
 
             // * Verify message on the main channel is from current user and properly formatted with lower opacity
             cy.get(`#post_${postId}`).should('have.class', 'current--user').within(() => {
                 cy.get('button').should('have.text', user1.username);
-                cy.get('p').should('have.text', 'test').and('have.css', 'color', 'rgba(61, 60, 64, 0.6)');
+                cy.get('p').should('have.text', message).and('have.css', 'color', 'rgba(61, 60, 64, 0.6)');
```

Comment:
Make use of random messages so that each test depends on it, rather than getting the previously posted (similar) "test" message.